adx/resource_adx_table_row_level_security_policy.go

```diff
@@ -0,0 +1,97 @@
+package adx
+
+import (
+	"context"
+	"fmt"
+
+	"encoding/json"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+type TableRowLevelSecurityPolicy struct {
+	IsEnabled bool
+	Query string
+}
+
+func resourceADXTableRowLevelSecurityPolicy() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceADXTableRowLevelSecurityPolicyCreate,
+		ReadContext:   resourceADXTableRowLevelSecurityPolicyRead,
+		DeleteContext: resourceADXTableRowLevelSecurityPolicyDelete,
+		UpdateContext: resourceADXTableRowLevelSecurityPolicyCreate,
```

Comment:
At least by azurerm convention - if we use the same func to do both Create and Update - func name is `resourceXXXCreateUpdate`. Could we amend that please?

Reply:
fixed